### PR TITLE
INT-3964: More `@EnableIntegration` documentation

### PR DESCRIPTION
**Pull request by artembilan**
JIRA: https://jira.spring.io/browse/INT-3964

### CODE REVIEW

**Comment by garyrussell** [comment]
Nice work! - Minor polishing here https://github.com/garyrussell/spring-integration/commit/fdea287e40639306cee8fdf0f1f8081f826e0e06

Pls review - it's late :smile:

**Comment by artembilan** [comment]
OK. Thank you for review and polishing.
That confirms that I go right way :smile: 
So, I'm applying your polishing and going ahead with the fix for XSDs as I promised in that new paragraph.

**Comment by artembilan** [comment]
Pushed.

File: src/reference/asciidoc/overview.adoc [hunk 1]
```diff
@@ -216,7 +216,7 @@ IMPORTANT: Starting with _version 4.0_, the `@EnableIntegration` annotation has
 registration of Spring Integration infrastructure beans (see
 http://docs.spring.io/spring-integration/docs/latest-ga/api/org/springframework/integration/config/EnableIntegration.html[JavaDocs]).
 This annotation is required when only Java & Annotation configuration is used, e.g. with Spring Boot and/or
-Spring Integration Messaging Annotation support and Spring Integration Java DSL with no XML integration configuration.
+Spring Integration Messaging Annotations support and Spring Integration Java DSL with no XML integration configuration.
```
Comment (garyrussell) [inline src/reference/asciidoc/overview.adoc]: You undid my polish :smile: and made it plural again.

In this context, it should be singular - don't ask to me to explain the quirks of English.

**Comment by garyrussell** [comment]
Wow - a lot of thankless work - but thank you anyway !!!

**Comment by garyrussell** [comment]
Merged as 66cd92a6e70fc96963fd35c04d97d99b7264e364

File: src/reference/asciidoc/overview.adoc (hunk 1 above)
Comment (artembilan) [inline src/reference/asciidoc/overview.adoc]: Yeah... Not sure that I got the rule (maybe magic word `support`) and won't make similar mistake in the future, but thank you for catching that up, fixing and merging.

**Pull request closed without merging**